stock_dynamic_routing_checkout_sync/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Camptocamp (https://www.camptocamp.com)
+{
+    "name": "Stock Routing Operations - Checkout Sync",
+    "summary": "Glue module",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": ["stock_dynamic_routing", "stock_checkout_sync"],
```

Comment:
Need to add `OCA/wms` in `oca_dependencies.txt` to get `stock_dynamic_routing`?

Reply:
added, thanks!